docs/drafts/developer.md

```diff
@@ -171,25 +148,33 @@ v0.33.1, built 2023-06-28
 (ctrl-c) to exit
 ```
 
-Typically, you'll want to press the space bar to have it open the UI in your web browser.
+At the end of the installation process, the command output will prompt you to press the space bar to open the web UI, which provides a useful overview of all the installed components.
+
+---
+
+## Special Setup for MacOS
+
+Some additional setup is necessary on Macintosh computers to install and configure compatible tooling.
 
-Shortly after starting, Tilt processes the `Tiltfile`, resulting in:
+### Install Homebrew and tools
+Follow the instructions to [install Homebrew](https://docs.brew.sh/Installation), and then execute the following command to install the required tools:
 
-- Building the go binaries
-- Building the images
-- Loading the images into kind
-- Running kustomize and applying everything except the Deployments that reference the images above
-- Modifying the Deployments to use the just-built images
-- Creating the Deployments
+```sh
+brew install bash gnu-tar gsed
+```
 
-### Making code changes
+### Configure your shell
+To configure your shell, either add this to your bash or zsh profile (e.g., in $HOME/.bashrc or $HOME/.zshrc), or run the following command in the terminal:
 
-Any time you change any of the files listed in the `deps` section in the `<binary name>_binary` `local_resource`,
-Tilt automatically rebuilds the go binary. As soon as the binary is rebuilt, Tilt pushes it (and only it) into the
-appropriate running container, and then restarts the process.
+```sh
+for bindir in `find $(brew --prefix)/opt -type d -follow -name gnubin -print`
```

Comment:
I noticed this command take a little while on my machine (likely I have a large home brew directory?)

I tweaked it like this to speed things up:
```suggestion
for bindir in `find $(brew --prefix)/opt -type d -follow -name gnubin -print -maxdepth 3`
```

Reply:
ah nice, adding that.

Reply:
Added to https://github.com/operator-framework/operator-controller/pull/1289